tasks/actorstate/actorstate.go

```diff
@@ -190,6 +198,8 @@ func (p *ActorStateProcessor) processActor(ctx context.Context, info ActorInfo)
 		return xerrors.Errorf("no extractor defined for actor code %q", info.Actor.Code.String())
 	}
 
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.TaskType, builtin.ActorNameByCode(info.Actor.Code)))
```

Comment:
would it be worth including this tag on the trace, or does that happen automagically?

Reply:
I can include it in the trace